akka-docs/src/main/paradox/stream/operators/Source/asSubscriber.md

```diff
@@ -4,12 +4,44 @@ Integration with Reactive Streams, materializes into a `org.reactivestreams.Subs
 
 @ref[Source operators](../index.md#source-operators)
 
-@@@ div { .group-scala }
 ## Signature
 
-@@signature [Source.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala) { #asSubscriber }
+@@@ div { .group-scala }
+
+@@snip[JavaFlowSupport.scala](/akka-stream/src/main/scala-jdk-9/akka/stream/scaladsl/JavaFlowSupport.scala) { #asSubscriber }
+
+@@@
+
+@@@ div { .group-java }
+
+@@snip[JavaFlowSupport.java](/akka-stream/src/main/java-jdk-9/akka/stream/javadsl/JavaFlowSupport.java) { #asSubscriber }
+
 @@@
 
 ## Description
 
-TODO: We would welcome help on contributing descriptions and examples, see: https://github.com/akka/akka/issues/25646
+If you want to create a @apidoc[Source] that gets its elements from another library that supports
+[Reactive Streams](https://www.reactive-streams.org/), you can use `JavaFlowSupport.Source.asSubscriber`.
+Each time this @apidoc[Source] is materialized, it produces a materialized value of type
+@javadoc[java.util.concurrent.Flow.Subscriber](java.util.concurrent.Flow.Subscriber).
+This @javadoc[Subscriber](java.util.concurrent.Flow.Subscriber) can be attached to a
+[Reactive Streams](https://www.reactive-streams.org/) @javadoc[Publisher](java.util.concurrent.Flow.Publisher)
+to populate it.
+
```

Comment:
don't we have to clarify the difference between JDK11 `JavaFlowSupport.Source.asSubscriber` and JDK8 `Source.asSubscriber`?

Reply:
Good point, added a note